test/commands/scanner/run.test.ts

```diff
@@ -754,12 +756,13 @@ describe('scanner:run', function () {
 				// there are five retries in addition to the initial run.
 				// Note: Please keep this up-to-date. It will make it way easier to debug if needed.
 				// The following categories are implicitly included:
-				// - 11 PMD categories
+				// - 8 PMD categories
 				// - 3 ESLint categories
 				// - 3 ESLint-Typescript categories
 				// - 1 RetireJS category
-				// For a total of 18
-				expect(implicitMessages || []).to.have.lengthOf(18, `Entries for implicitly added categories from all engines:\n ${JSON.stringify(implicitMessages)}`);
+				// - 2 Salesforce Graph Engine categories
```

Comment:
This is the only change to this file that isn't directly related to message sourcing.
Because `remove.test.ts` wasn't doing proper cleanup, some custom PMD jars were still registered when this test was executing, which meant there were more categories than there should have been.
Adding proper cleanup to those tests means that the number of expected categories actually went down.